### PR TITLE
Add instructions for Docker

### PR DESCRIPTION
**Pull request by Lakr233**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,6 +20,41 @@ TeleGPT is a Telegram bot based on [**teloxide**](https://github.com/teloxide/te
 
 ## Getting TeleGPT
 
+### Docker
+
+We recommend you to use the prebuilt docker image, it is published through GitHub Container Registry by GitHub Action. Currently, Linux & x86 machines are supported.
```
[the remark this answers is not in the document]
Comment (Lakr233) [reply README.md:25]: both are fine 🤪

[the remark this answers is not in the document]
Comment (Lakr233) [reply README.md:25]: Talking with author in WeChat, we need a discuss on container tagging.

[the remark this answers is not in the document]
Comment (unixzii) [reply README.md:25]: > both are fine 🤪

@Lakr233 I'm working on the final wording of README, and will update the PR later.

**Review by unixzii — APPROVED** [review]
It's ready to be merged.

**Pull request merged**